tests/compat/test_redirections.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+import pytest
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase, Client
+from django_webtest import DjangoTestApp
+
+from vitrina.compat.models import Redirections
+from vitrina.datasets.factories import DatasetFactory
+from django.test.utils import override_settings
+
+
+@pytest.mark.django_db
+def test_redirection_doesnt_exist(app: DjangoTestApp):
+    c = Client()
+    response = c.get('/neegzistuoja/')
+    assert response.status_code == 404
+
+
+@pytest.mark.django_db
+def test_redirection_exists(app: DjangoTestApp):
+    custom_dataset = DatasetFactory()
+    ctype = ContentType.objects.get(model='dataset')
+    Redirections.objects.create(created=datetime.now(),
```

Comment:
Do not ignore [warnings](https://github.com/atviriduomenys/katalogas/runs/8087420966?check_suite_focus=true#step:7:43), currently there are many warnings and using naive dates is one of the warnings.

```suggestion
    Redirections.objects.create(created=datetime.now(timezone.utc),
```

https://docs.python.org/3/library/datetime.html#aware-and-naive-objects